elastic/datadog_checks/elastic/metrics.py

```diff
@@ -258,6 +258,22 @@
     ),
 }
 
+JVM_METRICS_RATE = {
+    # Submit metrics as rate
+    'jvm.gc.collectors.young.rate': ('rate', 'jvm.gc.collectors.young.collection_count'),
```

Comment:
If `gc_collectors_as_rate` is enabled, `jvm.gc.collectors.young.collection_count` and `jvm.gc.collectors.young.rate` would both be submitted? Or just `jvm.gc.collectors.young.collection_count`?

Reply:
For backwards compatibility, both the rate and gauge versions would be submitted.